src/Microcosm.js

```diff
@@ -55,11 +55,17 @@ Microcosm.prototype = {
     return this
   },
 
+  shouldTransactionMerge() {
+    return true
+  },
```

Comment:
Default merger strategy. This should allow users to interact with undo history at a much higher level of abstraction. 

For example, if you wanted to only use the last 500 changes, you could do something like:

```
shouldTransactionMerge() {
    return this.history.path().length > 500
}
```

Based on some performance metrics, this will execute in `0.49ms`, which is fast enough for me.